### DISCUSSION TITLE
Twitter retweet tasks not working.

### DISCUSSION THREAD

**Discussion by darkagle901999** [IBAX-io/go-ibax#1813] [Q&A] [answered]
### Describe the bug

As you know, we were collecting points by retweeting. In this way, we would win an award. We trust this project. My score, which should have been in the 33000s, was low because of this mishap of the last day. I was in the first 1000, but with this job, all the balances changed. Some luckily did. But those of us who worked for 2 months could not do it. It is very sad, I hope this does not cause injustice.

### Reproduction

x

### System Info

```shell
x
```

### Logs

```shell
x
```

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't [already an issue](https://github.com/IBAX-io/go-ibax/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/IBAX-io/go-ibax/discussions) or join our [Discord Chat Server](https://discord.com/invite/zRX6Mwafya).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by Marco7235** [comment]
Either didnt I did tasks but didnt point

**Comment by ibaxio** [comment, marked as answer]
Twitter API has a limit of 2 million ...Each user request will cost 5 times and it took us 5 developer accounts over the last 3 days to keep this airdrop event going. We ran out of developer accounts. The airdrop event is over. Thank you all for your support and participation.

Please read https://twitter.com/IbaxNetwork/status/1488358542918623238
